docs/server/source/cloud-deployment-templates/client-tls-certificate.rst

```diff
@@ -24,38 +24,42 @@ Then :ref:`install and configure Easy-RSA in that directory <How to Install & Co
 Step 2: Create the Client Private Key and CSR
 ---------------------------------------------
 
-You can create the key and the CSR using:
+You can create the client private key and certificate signing request (CSR)
+by going into the directory ``client-cert/easy-rsa-3.0.1/easyrsa``
+and using:
 
-    .. code:: bash
+.. code:: bash
         
-        ./easyrsa init-pki
+   ./easyrsa init-pki
 
-        ./easyrsa gen-req bdb-instance-0 nopass
+   ./easyrsa gen-req bdb-instance-0 nopass
 
 
-Step 3: Sign the Client Certificate
------------------------------------
+You should change ``bdb-instance-0`` to a value based on the client
+the certificate is for.
```

Comment:
I prefer "should" because the word "could" will make them wonder if they should, which means we would have to give them a bunch more information about how to decide. Why not just stick with a convention of giving every instance of everything in the cluster its own unique name, even if that's not always strictly necessary?